test-harness/pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
           <groupId>net.sourceforge.htmlunit</groupId>
           <artifactId>htmlunit</artifactId>
-          <version>2.67.0</version>
+          <version>2.67.0</version><!--2.68.0 and 2.69.0 lead to compilation errors TODO: Investigate-->
```

Comment:
`2.69.0` works well for me. Could you please share which exact compilation error do you have? I guess I have solution for it.